src/expm.jl

```diff
@@ -52,12 +52,13 @@ end
     z = sqrt((a - d)*(a - d) + 4*b*c )
     e = exp((a + d - z)/2)
```

Comment:
we could compute expm1 here
```suggestion
    e = expm1((a + d - z)/2)
```

Reply:
I don't know how much `expm1` gets us here, but since the most common case where `z==0` is when we exponentiate the zeros matrix, I agree it makes sense to use it.